relayer/js/config.js

```diff
@@ -0,0 +1,17 @@
+require('dotenv').config();
+
+const envVars = [
+  'KOVAN_RPC_URL',
+  'MAINNET_RPC_URL',
+  'PRIVATE_KEY',
+  'PORT',
+  'MIN_TX_PROFIT',       // minimum profit per tx (in Wei) to consider submitting it
+  'GAS_PRICE'            // gas price of submitted txes (in Wei)
```

Comment:
I think this should be calculated beased upon GAS_PRICE = `Fee` - `MIN_TX_PROFIT`/ `Max_Gas_For_This_Tx`

Reply:
Ah interesting. I assume relayers would want to submit txes that have just a high enough gas price to be accepted within some time bound, but to your point, near that time bound I'm sure there's a relationship between gas price and how certain they want to be that the tx is confirmed quickly.

I think this will be something relayers will want fine grained control of when there are enough relayers so as to be competitive over withdraw proofs.